engine/src/mac-qt-player.mm

```diff
@@ -145,7 +158,54 @@ - (void)currentTimeChanged: (id)object
 }
 
 @end
+
+// PM-2015-05-27: [[ Bug 14349 ]] Dynamically subclass QTMovieView and override hitTest method to make mouse events respond to the superview
+@implementation com_runrev_livecode_MCQTKitHelper
+
+// We cannot subclass QTMovieView statically, because we have to dynamically load QTKit
+- (void) dynamicallySubclassQTMovieView:(QTMovieView *)p_qt_movie_view
+{
+    const char *t_prefix = "custom_dynamic_subclass_of_";
+    Class t_qt_movie_view_class = [p_qt_movie_view class];
+    NSString *t_qt_movie_view_class_name = NSStringFromClass(t_qt_movie_view_class);
```

Comment:
You can just use the constant string "com_runrev_livecode_MCQTMovieView" for the subclass name.